docs/authentication/hosted-add-authentication.mdx

```diff
@@ -183,9 +183,7 @@ https://auth-$REGION.beyondidentity.com/authenticator/v1/tenants/$TENANT_ID/real
 Upon successfully completing the UI flow, the Hosted Web Authenticator will redirect the user back to your application based on the provided **redirect_uri** in the authorization request appended with an **authorization code** and a **state**.
 
 ```bash title=302
-$(REDIRECT_URI)
-&code=$(AUTHORIZATION_CODE)
-&state=$(STATE)
+$(REDIRECT_URI)?code=$(AUTHORIZATION_CODE)&state=$(STATE)
```

Comment:
In order to get an id token, we need to pass a `scope` parameter of `openid`, so perhaps we should add that here?

Reply:
I'm in meetings for a couple of hours but will review this asap.

Reply:
Oh whoops, sorry, I didn't see which file I was commenting on. I thought I was up one. 😬